docs/app-center-preparing-assets.md

```diff
@@ -0,0 +1,124 @@
+---
+id: app-center-preparing-assets
+title: App Center
+sidebar_label: Preparing assets
+---
+
+
+##### Preparing assets for publishing your Aragon app
+
+Before you publish your Aragon app in the App Center, you will need to prepare a few types of assets to display it correctly, and differentiate from other apps.
```

Comment:
```suggestion
Before you publish your Aragon app in the App Center, you will need to prepare a few types of assets to display it correctly, and differentiate from other apps. Also, if you want your app to be part of the App Center contact us through [Aragon Chat](https://aragon.chat) to start a submission.
```

Reply:
Will write some docs on the submission process in another PR.